### ISSUE TITLE
Specify programming languages of developer

### ISSUE THREAD

**Issue by zdanek** [CodeMillApp/support#7]
Developer has to have ability to set known languages. I'm not interested in for example Ruby and Python but I'm getting offers for those languages.

**Comment by CodeMillApp** [comment]
Good idea. Do you think also a minimum payout should be added as a filter?

**Comment by zdanek** [comment]
Nice idea. That would fit more into programmers' needs.

**Comment by CodeMillApp** [comment]
@zdanek I am not sure if you were referring to the web app or the daily digest email that we send out. There was more interest in having this implemented as hot alerts for new tasks in your preferred languages. Just soft launched it, care to provide feedback? http://codemill.io/developers/alerts

btw, didn't implement the minimum payout just yet, one step at a time ;-)

**Comment by zdanek** [comment]
We're talking about the web app.
I've checked it and the panel works nice. 
Regarding to filtering I entered javascript and it found I open task at http://codemill.io/developers/tasks/open but then I've changed it to python and still I saw one javascript open task.

**Comment by CodeMillApp** [comment]
Ok, I took note. We'll add filtering in the web app in the future as well.
